src/plugins/plugin.legend.js

```diff
@@ -411,15 +412,30 @@ var Legend = Element.extend({
 			// Horizontal
 			var isHorizontal = me.isHorizontal();
 			if (isHorizontal) {
+				// default to center as this is the original behaviour of just position
+				var alignedX = ((legendWidth - lineWidths[0]) / 2) + labelOpts.padding;
+				if (opts.alignment === 'left') {
+					alignedX = 0;
+				} else if (opts.alignment === 'right') {
+					alignedX = ((legendWidth - lineWidths[0])) + labelOpts.padding;
+				}
 				cursor = {
-					x: me.left + ((legendWidth - lineWidths[0]) / 2) + labelOpts.padding,
+					x: me.left + alignedX,
 					y: me.top + labelOpts.padding,
 					line: 0
 				};
 			} else {
+				// default to top as this is the original behaviour of just position
+				var alignedY = me.top + labelOpts.padding;
+				var legendHeight = (fontSize + labelOpts.padding) * me.legendItems.length;
+				if (opts.alignment === 'bottom') {
+					alignedY = ((me.height - legendHeight));
```

Comment:
I don't think that `me.height` is guaranteed to be the bottom of the legend because it assumes that `me.top === 0` which isn't guaranteed. I would recommend adjusting this as `alignedY = me.bottom - legendHeight`